WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -45,29 +45,29 @@ def psych_sheet
   def psych_sheet_event
     @competition = competition_from_params
     @event = Event.find(params[:event_id])
-    preferred_format = @event.preferred_formats.first
+    @preferred_format = @event.preferred_formats.first
```

Comment:
@jfly This caused me some headaches because stuff broke after rebasing with `master`. Could you take a look at this commit: 405c27e361288b7c994b6bdc2e9fb674159118b6 and check whether I messed up some stuff.

btw. Was the renaming `position` to `pos` really necessary? It's gonna cause more conflicts unless I'll quickly merge this branch into master.